formatTest/unit_tests/expected_output/modules.re

```diff
@@ -399,7 +399,7 @@ module rec A: {
     | Leaf(string)
     | Node(ASet.t);
   let compare (t1, t2) =
-    switch ((t1, t2)) {
+    switch (t1, t2) {
```

Comment:
@let-def this is great.
So it was already valid syntax for the parser, but the pretty-printer was not using it.